clients/admin-ui/src/pages/management/cors-configuration.tsx

```diff
@@ -0,0 +1,172 @@
+      security: {
+        cors_origins: values.domains,
+      },
+    };
+    const result = await patchConfigSettingsTrigger(payload);
+
+    handleResult(result);
+  };
+
+  return (
+    <Layout title="CORS Configuration">
+      <Box data-testid="cors-configuration">
+        <Heading marginBottom={4} fontSize="2xl">
+          CORS Configuration
+        </Heading>
+        <Box maxWidth="600px">
+          <Text marginBottom={2} fontSize="md">
+            Add your CORS domains below
+          </Text>
+          <Text mb={10} fontSize="sm">
+            Please visit{" "}
+            <Link
+              color="complimentary.500"
+              href="https://docs.ethyca.com"
+              isExternal
+            >
+              docs.ethyca.com
+            </Link>{" "}
+            for more information on how to configure CORS domains.
+          </Text>
+        </Box>
+        <Box maxW="600px">
+          <FormSection title="CORS Domains">
+            <Formik<FormValues>
+              initialValues={corsDomains}
+              enableReinitialize
+              onSubmit={handleSubmit}
+            >
+              {({ dirty, values, isValid }) => (
+                <Form>
+                  <FieldArray
+                    name="domains"
+                    render={(arrayHelpers) => (
+                      <Flex flexDir="column">
+                        {values.domains.map((_: string, index: number) => (
+                          <Flex flexDir="row" key={index} my={3}>
+                            <CustomTextInput
+                              variant="stacked"
+                              name={`domains[${index}]`}
+                            />
+
+                            <IconButton
+                              ml={8}
+                              aria-label="delete-cors-domain"
+                              variant="outline"
+                              zIndex={2}
+                              size="sm"
+                              onClick={() => {
+                                arrayHelpers.remove(index);
+                              }}
+                            >
+                              <DeleteIcon />
+                            </IconButton>
+                          </Flex>
+                        ))}
+
+                        <Flex justifyContent="center" mt={3}>
+                          <Button
+                            aria-label="add-cors-domain"
+                            variant="outline"
+                            size="sm"
+                            onClick={() => {
+                              arrayHelpers.push("");
+                            }}
+                            rightIcon={<AddIcon />}
+                          >
+                            Add CORS domain
+                          </Button>
+                        </Flex>
+                      </Flex>
+                    )}
+                  />
+
+                  <Box mt={6}>
```

Comment:
the empty state looks a little weird since the save button appears. could we remove it if there are no domains?
![image](https://github.com/ethyca/fides/assets/24641006/90281d1c-d91d-4bb3-a066-64ee41f6862d)

Reply:
It does look a little weird but someone may want to remove all of they're CORS domains so removing it would prevent them from doing that.

Reply:
ah true!